stackslib/src/config/mod.rs

```diff
@@ -1210,53 +1210,321 @@ impl std::default::Default for Config {
 
 #[derive(Clone, Debug, Default, Deserialize, PartialEq)]
 pub struct BurnchainConfig {
+    /// The underlying blockchain used for Proof-of-Transfer.
+    /// Currently, only `"bitcoin"` is supported.
+    ///
+    /// Default: `"bitcoin"`
     pub chain: String,
+    /// The operational mode or network profile for the Stacks node.
+    /// This setting determines network parameters (like chain ID, peer version),
+    /// default configurations, genesis block definitions, and overall node behavior.
+    ///
+    /// Supported values: `"mainnet"`, `"mocknet"`, `"helium"`, `"neon"`, `"argon"`, `"krypton"`, `"xenon"`, `"nakamoto-neon"`.
```

Comment:
i'd argue to format this in a table (at least for now) because one of the questions that comes up is "what is xenon?"
perhaps something like this:
```
Supported values:
    mainnet => mainnet
    xenon => testnet
    mocknet => regtest
    ...
 ```
 i recognize it may be a bit messy - just trying to think of a way to make it more clear for users later on